src/components/Camera/Photo.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import * as React from 'react'
+import { h } from 'preact'
+
+import type { CameraType } from './CameraTypes'
+import classNames from 'classnames'
+import style from './style.css'
+import { screenshot } from '../utils/camera.js'
+
+import { CameraPure, CaptureActions } from './index.js'
+
+export default class Photo extends React.Component<CameraType> {
+  webcam = null
+
+  screenshot = () => screenshot(this.webcam, this.props.onScreenshot)
+  buttonText = () => {if (this.props.i18n) return this.props.i18n.t('capture.face.button')}
```

Comment:
Does this mean we don't have a fallback available when i18n isn't passed as a prop? It's not something that should happen (this particular component isn't exposed directly to any external developers I think?), so I don't think there's a change required. It's just to make sure I'm not missing anything.